webapp/packages/core-root/src/SessionEventSource.ts

```diff
@@ -76,10 +76,12 @@ export class SessionEventSource implements IServerEventEmitter<ISessionEvent, IS
 
     this.openSubject.subscribe(() => {
       this.onInit.execute();
+      this.disconnected = false;
```

Comment:
`disconnected` is internal flag that we need to prevent any in and out activity but it's for tracking current websocket status